build/steps-build-android.yml

```diff
@@ -56,14 +56,11 @@ steps:
       Copy-Item -Path '$(keyStore.secureFilePath)' -Destination '${{ parameters.pathToSrc }}\app\android\app\${{ parameters.androidKeystoreFile }}'
       Write-Host 'Key store copied to ${{ parameters.pathToSrc }}/app/android/app/${{ parameters.androidKeystoreFile }}.'
 
-#-if false
-# Workaround for the issue with the Flutter build runner not working sometimes.
 - task: FlutterCommand@0
   displayName: "Run Flutter Pub Get"
   inputs:
     projectDirectory: '${{ parameters.pathToSrc }}/app'
     arguments: 'pub get'
-#-endif
 
 - task: PowerShell@2
   displayName: 'Execute Package Rename'
```

Comment:
It feels like the `Execute Package Rename` step should not be present once the app is generated?

Reply:
Yes it will change the package name for Android which is different when in production.